lib/Db/CacheActorsRequest.php

```diff
@@ -66,6 +66,11 @@ public function __construct(
 	 * @return int
 	 */
 	public function save(Person $actor): int {
+		$source = $actor->getSource();
+		if (sizeof($source) >= CoreRequestBuilder::SOURCE_LENGTH) {
```

Comment:
I wonder if this can bring issues with MySQL's UTF8 vs UTF8mb4 ?

Reply:
sizeof is a wrapper for count which is an array function anyway, so use count instead of sizeof.

Further more you should use `strlen($source)` or `isset($source[CoreRequestBuilder::SOURCE_LENGTH])` instead.

Reply:
@tcitworld well, you can add unit tests to check this ;)

Reply:
@nickvergessen not the first time, don't know why I keep using sizeof()